configs/_base_/datasets/mot_challenge.py

```diff
@@ -0,0 +1,88 @@
+# dataset settings
+dataset_type = 'MOTChallengeDataset'
+data_root = 'data/MOT17/'
+
+# data pipeline
+train_pipeline = [
+    dict(
+        type='UniformSample',
+        num_ref_imgs=1,
+        frame_range=10,
+        filter_key_img=True),
+    dict(
+        type='TransformBroadcaster',
+        share_random_params=True,
+        transforms=[
+            dict(type='LoadImageFromFile'),
+            dict(type='LoadTrackAnnotations', with_instance_id=True),
+            dict(
+                type='RandomResize',
+                scale=(1088, 1088),
+                ratio_range=(0.8, 1.2),
+                keep_ratio=True,
+                clip_object_border=False),
+            dict(type='PhotoMetricDistortion')
+        ]),
+    dict(
+        type='TransformBroadcaster',
+        share_random_params=False,
+        transforms=[
+            dict(
+                type='RandomCrop',
+                crop_size=(1088, 1088),
+                bbox_clip_border=False)
+        ]),
+    dict(
+        type='TransformBroadcaster',
+        share_random_params=True,
+        transforms=[
+            dict(type='RandomFlip', prob=0.5),
+        ]),
+    dict(type='PackTrackInputs')
+]
+
+test_pipeline = [
+    dict(
+        type='TransformBroadcaster',
+        transforms=[
+            dict(type='LoadImageFromFile'),
+            dict(type='LoadTrackAnnotations', with_instance_id=True),
+            dict(type='Resize', scale=(1088, 1088), keep_ratio=True)
+        ]),
+    dict(type='PackTrackInputs')
+]
+
+# dataloader
+train_dataloader = dict(
+    batch_size=2,
+    num_workers=2,
+    persistent_workers=True,
+    # batch_sampler=dict(type='AspectRatioBatchSampler'), # used in image dataset: like LVIS # noqa: E501
```

Comment:
Add more comments to tell users this dataset is a video based dataset so we do not use AspectRatioBatchSampler